apps/examples/e2e/tests/test-text.spec.ts

```diff
@@ -156,7 +163,7 @@ test.describe('text measurement', () => {
 			measureTextSpansOptions
 		)
 
-		expect(formatLines(spans)).toEqual([['  ', 'testing', ' ', 'testing']])
+		expect(formatLines(spans)[0]?.[0]).toEqual('  ')
```

Comment:
Likewise, this fails all the time over the line splits, which isn't even what this test is testing.